sc_az_reindex_dag.py

```diff
@@ -1,14 +1,24 @@
-# Airflow DAG to index AZ Databases into Solr.
+"""Airflow DAG to index AZ Databases into SolrCloud."""
 from datetime import datetime, timedelta
 import airflow
+from airflow.hooks.base_hook import BaseHook
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
 from airflow.operators.python_operator import PythonOperator
-from cob_datapipeline.task_ingest_databases import ingest_databases
-from cob_datapipeline.task_ingest_databases import get_solr_url
-from cob_datapipeline.task_slackpost import task_az_slackpostonsuccess, task_slackpostonfail
 from cob_datapipeline.task_sc_get_num_docs import task_solrgetnumdocs
-from airflow.models import Variable
-from airflow.hooks.base_hook import BaseHook
-from tulflow.tasks import create_sc_collection, swap_sc_alias
+from cob_datapipeline.task_slackpost import task_az_slackpostonsuccess
+from tulflow import tasks
```

Comment:
```suggestion
from tulflow import tasks
import os
```

Reply:
This change is for adding other default env variables to what we pass into bash operator.

Reply:
@dkinzer I strongly believe you should not be overriding the Airflow BashOperator environment variables with the Python native Environment variables - because all this is doing is polluting the BashOperator enviornment with envvars from the airflow application level and not what is passed in. Why do you think we need to override the Operator here and pass in unnamed envvars?

Reply:
@cmharlow,  when we run the bash operator without overriding the env param, what get's automatically passed in is os.environment ... that happens later up in the stack...

@see `./airflow/operators/bash_operator.py:85:            self.env = os.environ.copy()`